queue/queue_test.go

```diff
@@ -14,6 +14,14 @@ import (
 )
 
 var server = "127.0.0.1:3013"
+var serversPool = []string{
```

Comment:
I see pool setup but don't see any tests with it.

Reply:
The pool used by queue/example_connection_pool_test.go